packages/ui-components/package.json

```diff
@@ -11,68 +11,46 @@
   "author": "Beyond Essential Systems <admin@tupaia.org> (https://beyondessential.com.au)",
   "source": "src/index.js",
   "main": "dist/index.js",
-  "module": "dist/index.esm.js",
-  "files": [
-    "lib"
-  ],
   "scripts": {
-    "build": "rollup -c",
-    "watch": "rollup -c -w",
-    "lint": "eslint --ignore-path ../../.gitignore .",
+    "build": "yarn package:build:js",
+    "lint": "yarn package:lint:js",
     "lint:fix": "yarn lint --fix",
     "storybook": "start-storybook -s public -p 6006",
-    "test": "jest --env=jsdom",
+    "test": "yarn package:test --env=jsdom",
     "test:coverage": "yarn test --coverage",
     "test:watch": "yarn test --watch"
   },
   "dependencies": {
+    "@date-io/date-fns": "1.x",
+    "@material-ui/core": "^4.9.8",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.47",
+    "@material-ui/pickers": "^3.2.10",
+    "@tupaia/utils": "1.0.0",
     "ace-builds": "^1.4.13",
     "date-fns": "^2.12.0",
+    "jsoneditor": "^9.0.0",
     "leaflet": "^1.7.1",
     "lodash.get": "^4.4.2",
     "lodash.throttle": "^4.1.1",
+    "moment": "^2.29.1",
+    "prop-types": "^15.7.2",
+    "react": "^16.13.1",
     "react-beautiful-dnd": "^13.1.0",
+    "react-dom": "^16.13.1",
     "react-leaflet": "^3.2.1",
+    "react-router-dom": "^5.1.2",
     "react-table": "^7.7.0",
     "recharts": "^1.8.5",
+    "styled-components": "^5.0.1",
     "xlsx": "^0.17.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.9.0",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.14.5",
-    "@date-io/date-fns": "1.x",
-    "@material-ui/core": "^4.9.8",
-    "@material-ui/icons": "^4.9.1",
-    "@material-ui/lab": "^4.0.0-alpha.47",
-    "@material-ui/pickers": "^3.2.10",
     "@material-ui/styles": "^4.9.10",
-    "@rollup/plugin-babel": "^5.3.0",
-    "@rollup/plugin-commonjs": "^18.0.0",
-    "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-multi-entry": "^4.0.0",
-    "@tupaia/utils": "1.0.0",
-    "axios": "^0.21.1",
+    "@storybook/react": "^6.3.9",
     "faker": "^4.1.0",
     "fast-glob": "^3.2.5",
-    "jsoneditor": "^9.0.0",
-    "moment": "^2.29.1",
```

Comment:
We had a setup I don't understand here where some dependencies were marked as `devDependencies` even though they were actually used throughout src. Not sure how it worked before. I moved these to normal `dependencies`, and the final build size of our frontends is the same, e.g. web-frontend was 20MB, and it is still 20MB.